store/sqlstore/upgrade.go

```diff
@@ -1304,5 +1306,15 @@ func upgradeDatabaseToVersion536(sqlStore *SqlStore) {
 }
 
 func hasMissingMigrationsVersion536(sqlStore *SqlStore) bool {
-	return !sqlStore.DoesColumnExist("SharedChannelUsers", "ChannelId")
+	if !sqlStore.DoesColumnExist("SharedChannelUsers", "ChannelId") {
+		return true
+	}
+	if !sqlStore.DoesColumnExist("SharedChannelRemotes", "LastPostUpdateAt") {
+		return true
+	}
+	if !sqlStore.DoesColumnExist("SharedChannelRemotes", "LastPostId") {
+		return true
+	}
```

Comment:
I am totally okay with keeping this for consistency but is there any practical difference between this and just running the `CreateColumnIfNotExists()` calls regardless of the version?

Reply:
Basically the idea is to make all the check in the same place and not split the "I need to run the migrations" knowledge between creating column, updating column, etc